snap/snapcraft.yaml

```diff
@@ -8,11 +7,17 @@ confinement: classic
 
 apps:
     heroku:
-        command: bin/heroku
+        command: bin/node $SNAP/lib/node_modules/heroku-cli/bin/run.js
 
 parts:
-  hello-node-snap:
+  heroku:
+    source: .
     plugin: nodejs
-    node-engine: 7.10.0
-    node-packages:
-      - heroku-cli
```

Comment:
oh good to know! so I can get rid of this? https://github.com/heroku/cli/blob/master/scripts/version#L6